repository/g.json

```diff
@@ -512,6 +512,17 @@
 			]
 		},
 		{
+			"name": "GitHub Color Scheme",
+			"details": "https://github.com/nicobeta/GitHub-Color-Scheme",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master",
```

Comment:
This line needs to be removed. You can only either have releases made from a branch, or tags. We only support tag-based releases for new packages.